src/arithmetic.jl

```diff
@@ -832,6 +901,63 @@ end
     return nothing
 end
 
+# In-place division and assignment: c = c / a
+# NOTE: Here `div!` *accumulates* the result of c / a in c[k] (k > 0)
+#
+# Recursion algorithm:
+#
+# k = 0: c[0] <-  c[0]/a[0]
+# k = 1: c[1] <- -c[1]
+#        c[1] <-  c[1] + c[0]*a[1]
+#        c[1] <- -c[1]/a[0]
+# k = 2: c[2] <- -c[2]
+#        c[2] <-  c[2] + c[0]*a[2] + c[1]*a[1]
+#        c[2] <- -c[2]/a[0]
```

Comment:
This resembles `fma`... maybe we should (in another PR) implement it

Reply:
I agree it's worth implementing!